pulp_ansible/app/galaxy/serializers.py

```diff
@@ -55,10 +58,20 @@ class GalaxyCollectionSerializer(serializers.Serializer):
     A serializer for a Collection.
     """
 
+    id = serializers.CharField(source="pulp_id")
     name = serializers.CharField()
-    namespace = serializers.CharField()
+    namespace = serializers.SerializerMethodField()
     href = serializers.SerializerMethodField(read_only=True)
     versions_url = serializers.SerializerMethodField(read_only=True)
+    created = serializers.DateTimeField(source="pulp_created")
+    modified = serializers.DateTimeField(source="pulp_last_updated")
+    deprecated = serializers.BooleanField()
```

Comment:
What happens when `deprecated` gets removed later? Do we need to add it in here at all now? Or keep it now and remove it later?

Reply:
I started this PR before looking CLI code, since I didn't see the use of `deprecated` I think we can remove it